RELEASING.md

```diff
@@ -0,0 +1,57 @@
+# Releasing container-diff
+
+The github.com/GoogleCloudPlatform/container-diff uses Container Builder triggers to build and release binaries.
+These triggers are setup via the Cloud Console, but the builds they execute live in this repo.
+
+## Continuous Builds
+
+Every commit to master is built and pushed automatically to a GCS location named via the COMMIT_SHA.
+
+```shell
+$ gsutil ls gs://container-diff/builds/
+gs://container-diff/builds/b726215b8b978e1d85257af5c2fd0a6fb8e116fd/
+
+$ gsutil ls 
+gs://container-diff/builds/b726215b8b978e1d85257af5c2fd0a6fb8e116fd/
```

Comment:
I think you want this at the end of the previous line instead of on a new one?

Reply:
good catch :)